itemadapter/src/main/java/com/mattskala/itemadapter/ItemAdapter.kt

```diff
@@ -38,6 +41,10 @@ open class ItemAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
             override fun getNewListSize(): Int {
                 return newItems.size
             }
+
+            override fun getChangePayload(oldItemPosition: Int, newItemPosition: Int): Any {
+                return newItems[newItemPosition]
```

Comment:
We could go further and introduce `getChangePayload` in `Item`, so that the user can create a change payload if needed. The default implementation would just return the new item.